frontend/components/product-list/sections/FeaturedProductListSection.tsx

```diff
@@ -140,7 +130,8 @@ export function FeaturedProductListSection({
                   apiKey={ALGOLIA_API_KEY}
                   initialIndexName={algoliaIndexName}
                   filtersPreset={
-                     productList.filters ?? `device:${productList.deviceTitle}`
+                     productList.filters ??
+                     `device:${productList.deviceTitle} AND public=1`
```

Comment:
If `productList.filters` is set, do we need to ensure that it gets a `AND public = 1` also?

Reply:
I can't think of any

Reply:
Oh I just misread the code. We tack `public=1` on _after_ that conditional is evaluated. Carry on :D

Reply:
> We tack `public=1` on _after_ that conditional is evaluated.

It doesn't look that way to me. I would also expect the filter for `public` to be part of the API key generated on the server side. Otherwise, clients are free to specify public/non-public as they like.

Reply:
I agree @sctice-ifixit but I think it's not using the securely generated key yet.